src/testRenderer.re

```diff
@@ -1,7 +1,7 @@
 type t;
 
 [@mel.module "react-test-renderer"]
-external create: React.element => t = "create";
```

Comment:
we can use `module React = React0` (or whatever future name we come up with).

We could also move TestRenderer and TestUtils to a different library `react.test` or something. Given that those aren't used that often, it could probably make sense to have them in a different library.

Reply:
`react-test-renderer` seems like a good library name